modules.d/95iscsi/module-setup.sh

```diff
@@ -120,8 +121,14 @@ install_iscsiroot() {
     local_address=$(ip -o route get to $iscsi_address | sed -n 's/.*src \([0-9a-f.:]*\).*/\1/p')
     ifname=$(ip -o route get to $iscsi_address | sed -n 's/.*dev \([^ ]*\).*/\1/p')
 
-    #follow ifcfg settings for boot protocol
-    bootproto=$(sed -n "/BOOTPROTO/s/BOOTPROTO='\([[:alpha:]]*6\?\)4\?'/\1/p" /etc/sysconfig/network/ifcfg-$ifname)
+    # follow ifcfg settings for boot protocol
+    for _path in \
+        "/etc/sysconfig/network-scripts/ifcfg-$ifname" \
+        "/etc/sysconfig/network/ifcfg-$ifname" \
+    ; do
+        [ -f "$_path" ] && bootproto=$(sed -n "s/BOOTPROTO='\?\([[:alpha:]]*6\?\)4\?/\1/p" "$_path")
```

Comment:
This should be probably `-r`:
```suggestion
        [ -r "$_path" ] && bootproto=$(sed -n "s/BOOTPROTO='\?\([[:alpha:]]*6\?\)4\?/\1/p" "$_path")
```